setup.cfg

```diff
@@ -0,0 +1,37 @@
+[metadata]
+name = xroms
+description = Commonly used scripts for working with ROMS output in xarray
+author = Rob Hetland
+url = https://github.com/hetland/xroms
+long_description_content_type = text/markdown
+long_description = file: README.md
+license = MIT
+license_file = LICENSE
```

Comment:
Got most of the metdata here from the README. Let me know if you want to add more.